### PR TITLE
refactor: update learn callout

### PR DESCRIPTION
**Pull request by kendallstrautman**
- Removes local learn callout component for the recently published [react-components package](https://react-components.vercel.app/?component=LearnCallout).
- Adjusts the homepage icons to have a flat background / black icons. 
Screen Shot 2021-04-09 at 12 17 55 PM
- Adds a temporary override on downloads page so the contrast is readable. 
Screen Shot 2021-04-09 at 12 15 42 PM

### CODE REVIEW

File: website/pages/community/style.css [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #p-community {
+  --brand-link: var(--nomad-link);
```
Comment (zchsh) [inline website/pages/community/style.css:2]: Thanks for picking this up 💖 🧙 

I have a disgustingly minor nitpick ahah 😜 The underline is, unfortunately, not `text-decoration: underline` but an `::after` pseudo-element that uses the `--brand` property. And I think it's still blue.

So I think we should also set `--brand` to `--nomad-link` to fix that:

```suggestion
  --brand-link: var(--nomad-link);
  --brand: var(--nomad-link);
```

To be clear, I think this is a great temporary fix 0 even with the blue underline, I don't know that it's really that noticeable, it just bothered me 😬 

### Before this change

blue-underline

### After this change

nomad-underline

---

Also just a big +1 to fixing this in the component as you mentioned on [the Asana task](https://app.asana.com/0/1100423001970639/1200149091112826/f) by accepting a `product` prop. Added a note on the potential switch to `text-decoration: underline` as well 👍.

File: website/pages/downloads/style.css [hunk 2]
```diff
@@ -1,4 +1,15 @@
 #p-downloads {
   margin-top: 72px;
   margin-bottom: 72px;
+
+  /** DEBT - temporary override until product download page is rolled out */
```
Comment (zchsh) [inline website/pages/downloads/style.css:5]: Added a note on this to [the Nomad task on the Downloads rollout board](https://app.asana.com/0/1100423001970639/1199637416872535/f) - I think it's great we're meeting the contrast reqs, this feels like a good compromise to me ❇️

File: website/pages/home/img/why-nomad/flexible-workload-support.svg [hunk 3]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 70.16 70.16"><defs><linearGradient id="linear-gradient" y1="35.08" x2="70.16" y2="35.08" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="var(--nomad, #60dea9)"/><stop offset="1" stop-color="#60dea9"/></linearGradient></defs><title>15</title><rect width="70.16" height="70.16" rx="20" style="fill:url(#linear-gradient)"/><path d="M28.63,20.06H23.89a3.16,3.16,0,0,0-3.15,3.16V28" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M49.15,28V23.22A3.16,3.16,0,0,0,46,20.06H41.26" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M41.26,48.47H46a3.15,3.15,0,0,0,3.16-3.15V40.58" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M20.74,40.58v4.74a3.15,3.15,0,0,0,3.15,3.15h4.74" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/></svg>
```
Comment (zchsh) [inline website/pages/home/img/why-nomad/flexible-workload-support.svg:1]: Making this comment so I can link to it in [the SVG inline styling cleanup task in Asana](https://app.asana.com/0/1100423001970639/1200120445315586/f).

**Review by zchsh — APPROVED** [review]
✅ Awesome 🎉  I reviewed all the affected pages ([home](https://nomad-3t23kn81n-hashicorp.vercel.app), [downloads](https://nomad-3t23kn81n-hashicorp.vercel.app/downloads),  [community](https://nomad-3t23kn81n-hashicorp.vercel.app/community)) pretty thoroughly, obvs with a focus on the affected components, and they all look great.

Code also looks great to me.

I had a SUPER minor nitpick around the underline colour on the Community page. Feel free to ignore, I certainly don't think it should block shipping this if you want to keep moving 👍

File: website/pages/community/style.css (hunk 1 above)
Comment (kendallstrautman) [reply website/pages/community/style.css:2]: Good catch! Thank you

**Pull request merged**